infra/capi/cluster-capi-infrastructure-ironic/cluster-api-infrastructure-ironic.yaml

```diff
@@ -0,0 +1,210 @@
+apiVersion: v1
+data:
+  CACHEURL: http://172.22.0.1/images
+  DEPLOY_KERNEL_URL: http://172.22.0.2:6180/images/ironic-python-agent.kernel
+  DEPLOY_RAMDISK_URL: http://172.22.0.2:6180/images/ironic-python-agent.initramfs
+  DHCP_HOSTS: b4:96:91:c0:31:64,id:*;b4:96:91:c4:e3:f4,id:*
+  DHCP_IGNORE: tag:!known
+  DHCP_RANGE: 172.22.0.10,172.22.0.100
+  DNS_IP: 172.22.0.3
+  GATEWAY_IP: 172.22.0.4
+  HTTP_PORT: "6180"
+  IRONIC_BASE_URL: http://172.22.0.2:6385
+  IRONIC_ENDPOINT: http://172.22.0.2:6385/v1/
+  IRONIC_INSPECTOR_VLAN_INTERFACES: all
+  IRONIC_KERNEL_PARAMS: console=ttyS0
+  IRONIC_RAMDISK_SSH_KEY: PLACEHOLDER_SSH_KEY
+  PROVISIONING_INTERFACE: eth2
+  USE_IRONIC_INSPECTOR: "false"
+kind: ConfigMap
+metadata:
+  name: baremetal-operator-ironic-bmo-configmap-6cf9t7484b
+  namespace: baremetal-operator-system
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: baremetal-operator-ironic
+  namespace: baremetal-operator-system
+spec:
+  minReadySeconds: 10
+  replicas: 1
+  selector:
+    matchLabels:
+      name: ironic
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        name: ironic
+    spec:
+      containers:
+      - command:
+        - /bin/rundnsmasq
+        envFrom:
+        - configMapRef:
+            name: baremetal-operator-ironic-bmo-configmap-6cf9t7484b
+        image: quay.io/metal3-io/ironic
```

Comment:
Let us point to specific image version instead of latest which is always moving target. I guess 29.0 was recent released version. Same change needed for all containers

Reply:
Good catch, This file is generated from the command `kustomize build ironic-deployment/default > cluster-api-infrastructure-ironic.yaml`, let me ask the ironic K8S community how they handle image versions.